server/api-util/lineItemHelpers.js

```diff
@@ -20,12 +20,15 @@ const LINE_ITEM_DAY = 'line-item/day';
  * @returns {Money} lineTotal
  */
 exports.calculateTotalPriceFromQuantity = (unitPrice, unitCount) => {
-  const numericPrice = convertMoneyToNumber(unitPrice);
-  const numericTotalPrice = new Decimal(numericPrice).times(unitCount).toNumber();
-  return new Money(
-    convertUnitToSubUnit(numericTotalPrice, unitDivisor(unitPrice.currency)),
-    unitPrice.currency
-  );
+  const amountFromUnitPrice = getAmountAsDecimalJS(unitPrice);
+
+  // NOTE: We round the total price to the nearest integer.
+  //       Payment processors don't support fractional subunits.
+  const totalPrice = amountFromUnitPrice.times(unitCount).toNearest(1, Decimal.ROUND_HALF_UP);
```

Comment:
You're the best to comment on style issues, but should these be on newlines as below?

Reply:
@sktoiva Prettier...